::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Wed, 4 Apr 2018 20:08:55 +0200
Subject: [PATCH] linux.system.repo: Drop arch for openstack repo

Now that mcp-repos mirror has arm64 packages too, we can either extend
the current architecture list or just drop it and let APT figure it
out.

Extending the arch list to 'amd64,arm64' would lead to doubling repo
metadata downloads on all archs, while dropping it will allow APT to
only download the repo metadata for the current system arch.

To keep old behavior for amd64 and prevent multiple downloads, let's
just drop the 'architectures' field.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/system/repo/mcp/openstack.yml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/linux/system/repo/mcp/openstack.yml b/linux/system/repo/mcp/openstack.yml
index 465b75d0..8de17774 100644
--- a/linux/system/repo/mcp/openstack.yml
+++ b/linux/system/repo/mcp/openstack.yml
@@ -7,7 +7,6 @@ parameters:
       repo:
         mirantis_openstack:
           source: "deb http://mirror.fuel-infra.org/mcp-repos/${_param:openstack_version}/${_param:linux_system_codename} ${_param:openstack_version} main"
-          architectures: amd64
           key_url: "http://mirror.fuel-infra.org/mcp-repos/${_param:openstack_version}/${_param:linux_system_codename}/archive-mcp${_param:openstack_version}.key"
           pin:
           - pin: 'release l=${_param:openstack_version}'
